### PR TITLE
add configurable env for GITHUB_ROOT

### PR DESCRIPTION
**Pull request by jessicaxiejw**
Adding an environment variable called `DEV_GITHUB_ROOT` so that the GITHUB_ROOT is configurable and isn't always hard coded to `~/src/github.com`.

### CODE REVIEW

File: lib/dev/commands/cd.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ module Commands
       basename = RUBY_PLATFORM =~ /darwin/ ? 'fzy_darwin' : 'fzy_linux'
       File.expand_path("vendor/#{basename}", ROOT)
     end
-    GITHUB_ROOT = '~/src/github.com'
+    GITHUB_ROOT = ENV.fetch('DEV_GITHUB_ROOT', '~/src/github.com')
```
Comment (ajkerr) [inline lib/dev/commands/cd.rb:9]: There's another reference to `~/src/github.com` in `clone.rb` that you probably should handle as well.

Comment (jessicaxiejw) [reply lib/dev/commands/cd.rb:9]: 🤔 This just gets more complicated. If I want to DRY the code, there is a philosophical debate between using env or some kinds configuration file (eg/ yaml).

What are your thoughts on creating a new module for configuration and parse the github_root from there. So something like

```ruby
# in lib/dev/config.rb

module Dev
  module Config
     def self.github_root
        # read from env example:
           ENV.fetch('DEV_GITHUB_ROOT', '~/src/github.com')

        # or parse some kind of yaml file and load the key here
     end
  end
end
```

I am personally OK with using environment variable to keep it simple. However, we could have something like `~/.dev/config.yaml`.

Comment (jessicaxiejw) [reply lib/dev/commands/cd.rb]: That's a good point about dev config set.

I have updated the code, not sure if @burke is OK with the direction.

**Review by ajkerr — APPROVED** [review]
Looks good, thank you!

**Pull request merged**